test/features/prepare_config.feature

```diff
@@ -0,0 +1,25 @@
+# Date:  08/07/2020
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+Feature: Prepare the skos-history config file and folder structure
+
+  As a user,
+  I want to structure the dataset folder and generate the skos-history configuration file from a minimal set of parameters,
+  so that I can execute the skos-history.
+
+  Scenario: Generating the skos-history config file
+    Given mandatory descriptive metadata
+    And the root path of folder structure
+    When the user runs the config generator
+    Then a correct configuration file is created in the folder structure
+
+  Scenario: Set up the folder structure
```

Comment:
I would put this `Scenario` first: you first prepare the folder structure, and then generate the `skos-history` config file. 
(this is only for readability purposes)

Reply:
agree. you can swap the order.